packages/apps/spaces/ui/presentation/Table/TActions.tsx

```diff
@@ -0,0 +1,51 @@
+import { forwardRef, useRef } from 'react';
+import type { MenuProps } from 'primereact/menu';
+
+import { IconButton } from '@spaces/atoms/icon-button';
+import { EllipsesV } from '@spaces/atoms/icons';
```

Comment:
let's avoid barrel imports until we solve tree shaking issue

Reply:
sure